index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+
+      <a href="#welcome-section">Welcome section</a>
+      <a href="#projects">Projects section</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Hey I am Denis</h1>
+    </section>
+
+    <a href="https://github.com/dd-obua" id="profile-link">Git Hub</a>
+    
+    <section id="projects" style="margin-top: 900px">
+      <div class="project-tile">
+        clwmvelkvnlkevnk nskn lkanvk oan
```

Comment:
- [ ] Again, this message is really vague, you should explain in details what the requested change is. Saying something more significant is needed is misleading, you could phrase it like `I think it would be better to have meaningful text in this section, please change the text.`

Reply:
- [ ] Weldone updating this comment based on the previous reviewer's comment. You can also add that meaningful text will make it easier for you and browser to read your website, also makes the website more accessible and readable by screen readers and improve search engine optimization